src/main/java/dinkplugin/notifiers/BaseNotifier.java

```diff
@@ -25,7 +27,11 @@ public abstract class BaseNotifier {
     private DiscordMessageHandler messageHandler;
 
     public boolean isEnabled() {
-        return !WorldUtils.isIgnoredWorld(client.getWorldType()) && settingsManager.isNamePermitted(client.getLocalPlayer().getName());
+        Set<WorldType> world = client.getWorldType();
+        if ((config.ignoreSeasonal() && world.contains(WorldType.SEASONAL)) || WorldUtils.isIgnoredWorld(world)) {
+            return false;
+        }
```

Comment:
Can we split this up into two separate checks? 1 for the ignore seasonal & 1 for the isIgnoredworld